### PR TITLE
version 2.2.2

### PR DESCRIPTION
**Pull request by leewang0**
# Version 2.2.2

## Date: 2022-09-30

- Fix paths for shared passthrough models
- Do not attempt to serialize empty lists as null objects
- Fix datetime() default breaking sync status endpoints
- Improve expands model attribute handling
- Remove date and datetime from Any type
- Various model updates

### Note

There was a small typo in the prior commit which renamed this package in pip, so we keep version 2.2.2 to keep it sequential.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """
-    Merge Ticketing API
```
Comment (leewang0) [inline setup.py:2]: Typo fixes start here

File: MergePythonSDK/shared/__init__.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # flake8: noqa
 
 """
-    Merge Unified API
+    Merge Ticketing API
```
Comment (conorbergman) [inline MergePythonSDK/shared/__init__.py:4]: Switch back to Unified (and likewise below)

File: setup.py (hunk 1 above)
Comment (conorbergman) [reply setup.py:2]: a mustache dispair, ty for fix!

**Comment by leewang0** [comment]
Too many unintentional changes

**Pull request closed without merging**